<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageEngines</class>
 <widget class="QWidget" name="ManageEngines">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Engines</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QListWidget" name="lst_engines"/>
     <widget class="QWidget" name="pnl_options" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_add">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_del">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>released()</signal>
   <receiver>ManageEngines</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_add</sender>
   <signal>released()</signal>
   <receiver>ManageEngines</receiver>
   <slot>_add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_del</sender>
   <signal>released()</signal>
   <receiver>ManageEngines</receiver>
   <slot>_delete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_add()</slot>
  <slot>_delete()</slot>
 </slots>
</ui>
